api/envoy/api/v2/listener.proto

```diff
@@ -4,6 +4,7 @@ package envoy.api.v2;
 
 import "envoy/api/v2/core/address.proto";
 import "envoy/api/v2/core/base.proto";
+// buf:lint:ignore IMPORT_USED
 import "envoy/api/v2/core/socket_option.proto";
```

Comment:
im not sure why the updated `buf` wants to get rid of these - they seem to be used below - either way adding an ignore works around

Reply:
argg that didnt work proto_format wants to remvoe the lint exceptions

Reply:
i shifted the ignores to the config file